diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -42,6 +51,32 @@ open class DiktatJavaExecTaskBase @Inject constructor(
     @get:Internal
     internal var shouldRun = true
 
+    /**
+     * Files that will be analyzed by diktat
+     */
+    @get:IgnoreEmptyDirectories
+    @get:SkipWhenEmpty
+    @get:PathSensitive(PathSensitivity.RELATIVE)
+    @get:InputFiles
+    val actualInputs: FileCollection by lazy {
+        with(diktatExtension) {
+            // validate configuration
+            require(inputs == null && excludes == null) {
+                "`inputs` and `excludes` for diktat should be changed for inputs(Action)"
```

Comment:
"`inputs` and `excludes`  arguments for diktat task are deprecated and now should be changed for inputs(Action). Please check the Readme.md for more info"